CommandHelper/LocalPackages/commands/buffs/doctordoom.ms

```diff
@@ -0,0 +1,28 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§2D§ao§2ct§ao§2r §aD§2o§ao§2m',
+            'class': 'doctordoom',
+            'sig': 'rteoA/8KMAXQ4pc2r1yarMDmdG7QKS5X1ru4SuY5D9uoB+hwzgEw1baIuveh7Qwd/ddhUqydlGUYtuCySXrPB3sbKSCwSHbiNN/NP/GtmFE6IHDC8r1sdknIVQhXk58uRXgrD3xNmU5K6+O2kOlgAVplCRfcvH2Wvn8aB/ShXZxiuKiMk3DevZOMquF4/fiqUQDhsVg8mex/j1zwUEUm+G2ckT/17WZ+VYkhxcEgovrJYZ+n6hl7SYeQ4n5RxM7bIgOT8Q4abtN1c4Kru3cOdEvlPak/ZrPppZK7qo+ZY9F9s6W8mpV06Prf0ZgHaN6EoHBMyd0eWwAik53NEQVa8En6sRl9/AkCuxmOGRC/MXbl37FPKFga2wCrXAHkjftZuJexbydUtgvrDomAFoBHFkrxfWcWvXjYNc4z6eAA2COWKiVw2T+umh8Acph5Gcf1+/7p9esD6GVVNyjmsCxPK5JBqMLLfXrx4C/Be6j91i+h1K9tNNkIhL6Uq1JlAirrUe2O75Dw8vhmiYVRQy9MdgkS/CAg5cpuM6+NdlpYfGcStCWzyehr1M9ZMPS/I6HXggnqx7xjeCFT8cB/MNzni8uN8sD07EnzG74ifgyHC2jKkG2e3Fwm19CCBVdeSdIuvZOVAZCmVtkswFjPlEPvWdq7W/7pKxhJGv+aAKDx714=',
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODQ1ODkyOTg4MTQsInByb2ZpbGVJZCI6IjQwODdiYzdmYzIxMzQ3ZGZiZjJkNDE2NTAwMzBiZTQ1IiwicHJvZmlsZU5hbWUiOiJVbHRyYU93bERDIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83NDczOGQ3OWFiY2ZjNDg5NjdmOWRkNjUzYjlkNDBiZGIxNDc0Yzk1NWJkZGRiY2RhNGFhNDI1ZGUyZTZiZDI3In19fQ==',
+            'trail': '',
+            'op': 'true',
+            'speech': "Life is a game of chess.",
+            'powers': array(
+                "/buff",
+                "/kill",
+                "/skull <player>"
+                "/hover on",
+                "/hover off",
+                "/empower",   
+                "/gmc",
+                "/gms"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 15 true",
+                "/effect ". @plr ." 10 9999 4 true",
+                "/effect ". @plr ." 11 9999 4 true"
```

Comment:
https://video.search.yahoo.com/yhs/search;_ylt=AwrTcdBiKH1YSyAALxQPxQt.?p=Dr.Doom+exploding+heads+in+Fantastic+Four+movie&fr=yhs-omr-001&fr2=piv-web&hspart=omr&hsimp=yhs-001&type=86311436#action=view&id=5&vid=4af20d00b907867b08585f2441bc282a